packages/yuudachi/locales/en-US/translation.json

```diff
@@ -124,25 +125,59 @@
 			},
 			"anti_raid_nuke": {
 				"errors": {
+					"invalid_file_type": "Invalid file type provided, must be a `.txt` or `.json` file.",
+					"invalid_avatar": "Invalid avatar format, must be an raw hash, a userId, a URL or `none`.",
 					"no_hits": "The Anti-Raid-Nuke will not hit anyone, you might want to re-adjust the parameters and try again!",
-					"parameters": {
-						"heading": "Parameters:",
-						"current_time": "• Current time: {{- now}}",
-						"join_after": "• Join date after: {{- join}}",
-						"created_after": "• Account creation after: {{- age}}",
-						"pattern": "• Username matching pattern:",
-						"avatar": "• Avatar hash (user or member) matching: {{- avatar}}",
-						"days": "• Deleting messages sent in the last {{count}} day",
-						"days_plural": "• Deleting messages sent in the last {{count}} days",
-						"days_none": "• Not deleting any messages"
+					"no_hits_file": "The Anti-Raid-Nuke will not hit anyone, content you provided did not contain any valid Member Ids!",
```

Comment:
```suggestion
					"no_hits_file": "The Anti-Raid-Nuke will not hit anyone, content you provided did not contain any valid ids.",
```

Reply:
Changed in [801e5f6](https://github.com/Naval-Base/yuudachi/pull/1069/commits/801e5f629bf9e6249b6942ce8c7c3a09b006acbb)